src/apps/ocioview/main.py

```diff
@@ -30,12 +30,11 @@ def excepthook(exc_type, exc_value, exc_tb):
     sys.excepthook = excepthook
 
     # OpenGL core profile needed on macOS to access programmatic pipeline
-    gl_format = QtOpenGL.QGLFormat()
-    gl_format.setProfile(QtOpenGL.QGLFormat.CoreProfile)
-    gl_format.setSampleBuffers(True)
+    gl_format = QtGui.QSurfaceFormat()
+    gl_format.setProfile(QtGui.QSurfaceFormat.CoreProfile)
     gl_format.setSwapInterval(1)
```

Comment:
One thing to note, in PR #1875 I modify `QSurfaceFormat` in addition to the original `QOpenGLFormat`, changing the swap interval to 0 (turning off v-sync), as I found having a positive value with a `Qt3DWindow` in the app was lowering `QGraphicsView` frame rate (i.e. the presence of the cube inspector was reducing the curve inspector's performance, which this change fixed).

Reply:
Anything I do need to update in this PR? Also out of curiosity, did you try to use [triple buffering](https://doc.qt.io/qt-6/qsurfaceformat.html#SwapBehavior-enum)?

Reply:
No change is needed in this PR. I just wanted to mention it in case you had thoughts. I have not tried triple buffering, but that's a great idea. Thanks.